internal/pkg/model/manifest.go

```diff
@@ -21,11 +21,11 @@ type RecordPaths interface {
 	// GetRelativePath - for example path of the object inside parent object/path.
 	GetRelativePath() string
 	// SetRelativePath - for example path of the object inside parent object/path.
-	SetRelativePath(string)
+	SetRelativePath(p string)
```

Comment:
I'd rather use actual words than abbreviations to be honest... like `path` here.

Reply:
06c930872aa366ca9896ba7c9a5b26ef4293aefa

Reply:
You still left `p` here... :sweat_smile:

Reply:
You still left `p` here... :sweat_smile:

```suggestion
	SetRelativePath(path string)
```